pcsx2/R5900OpcodeImpl.cpp

```diff
@@ -983,7 +983,26 @@ void SYSCALL()
 			else
 				__Deci2Call(cpuRegs.GPR.n.a0.UL[0], (u32*)PSM(cpuRegs.GPR.n.a1.UL[0]));
 		}
-		break;
```

Comment:
I think `break;` shouldn't have been removed?

Reply:
You're right, that's what I get for writing this late at night.